tenderly/utils/types.ts

```diff
@@ -0,0 +1,30 @@
+export type EstimateGasParams = {
+  from?: string
+  to: string
+  gas?: number
+  gasPrice?: number
+  value?: number
+  data?: string
+}
+
+export type Transaction = {
```

Comment:
Can leverage web3's `TransactionOutput` and `TransactionInfo` if you feel like it

https://github.com/web3/web3.js/blob/7c13a90e6a1bdffbcce8f9a2dc0d2272b9be5e4e/packages/web3-types/src/eth_types.ts#L80-L94

https://github.com/web3/web3.js/blob/7c13a90e6a1bdffbcce8f9a2dc0d2272b9be5e4e/packages/web3-types/src/eth_types.ts#L363-L369